requirements.txt

```diff
@@ -0,0 +1,7 @@
+lxml
+bleach==1.4.2
+oauthlib==1.0.3
+Mako
+git+https://github.com/edx/XBlock.git#egg=XBlock
```

Comment:
@doctoryes @nedbat how do we version XBlock right now? Do we have anything in place where we would guarantee functionality between versions?